ansible/pbTestScripts/buildJDK.sh

```diff
@@ -15,7 +15,7 @@ export TARGET_OS=linux
 export ARCHITECTURE=x64
 export JAVA_TO_BUILD=jdk8u
 export VARIANT=openj9
-export JDK7_BOOT_DIR=/usr/lib/jvm/java-1.7.0
+export JDK7_BOOT_DIR=$(find /usr/lib/jvm/ -name java-1.7.0-\*)
```

Comment:
On my C6 system that would pick up at least two entries which would make a mess of the `JDK7_BOOT_DIR` variable (since there is one symlink to something else,. both matching that pattern)

```
[root@159 jvm]# ls -ld java-1.7.0-*
drwxr-xr-x 3 root root 4096 May  3  2018 java-1.7.0-openjdk-1.7.0.171.x86_64
drwxr-xr-x 3 root root 4096 Aug 10  2018 java-1.7.0-openjdk-1.7.0.181.x86_64
drwxr-xr-x 2 root root 4096 Nov 21  2018 java-1.7.0-openjdk-1.7.0.191.x86_64
drwxr-xr-x 2 root root 4096 Apr 23  2019 java-1.7.0-openjdk-1.7.0.211.x86_64
drwxr-xr-x 7 root root 4096 Oct 24 01:00 java-1.7.0-openjdk-1.7.0.241.x86_64
lrwxrwxrwx 1 root root   35 Oct 24 01:00 java-1.7.0-openjdk.x86_64 -> java-1.7.0-openjdk-1.7.0.241.x86_64
```

Reply:
New commit in that will look for the CentOS symlink first, and then look for the Ubuntu symlink after. If it can't find it, then the variable remains blank and it will default to building JDK8 with JDK8 - for instance in the case of Ubuntu1804